fixtures/cli_input/src/some_dir/mod.rs

```diff
@@ -0,0 +1,20 @@
+#[test]
+fn some_test() {
+    let mut n = 0;
+    n += 1;
+    some_noop();
+}
+
+#[allow(dead_code)]
+fn some_noop() {}
+
+#[test]
+fn unused_test() {
+    let value = unused_expression();
+    assert!(value);
+}
+
+#[allow(dead_code)]
+fn unused_expression() -> bool {
+    true
+} 
```

Comment:
```suggestion
```
Same comment as above: if you didn't have a specific reason for including these, let's please remove them.